yaml_edit/test/random_test.dart

```diff
@@ -54,8 +55,13 @@ dev_dependencies:
 ''');
 
       for (var j = 0; j < modificationsPerRound; j++) {
-        expect(
-            () => generator.performNextModification(editor), returnsNormally);
+        runZoned(() {
+          expect(
+              () => generator.performNextModification(editor), returnsNormally);
+        },
+            zoneSpecification: ZoneSpecification(
```

Comment:
Again, let's add a comment that we're doing this hide warnings from `package:yaml`.

Reply:
Also, can we run a single check that this won't eat errors / exceptions? (just a manual check will do, hehe)

From reading: https://api.dart.dev/stable/2.9.0/dart-async/runZoned.html
I don't think this is an error-zone, and only error-zones prevents errors from propagating to the calling context.